### DISCUSSION TITLE
ERREUR de téléchargement

### DISCUSSION THREAD

**Discussion by Franck-DUBUS** [aclap-dev/video-downloadhelper#742] [Help and Q&A]
### Your question

Mardi 28.05.24.
Bonjour ;

Je n'y comprends plus rie, depuis pas mal de temps, je n'arrive plus à télécharger avec Vidéo Download Internet de Firefox, en effet, j'ai toujours cette erreur à chaque fois ::

SideDownload error: 
[libaom-av1 @ 0000023d6d6eb440] Failed to decode frame: Bitstream not supported by this decoder

    Last message repeated 30 times

[vost#0:1/copy @ 0000023d6d765140] Error submitting a packet to the muxer: Invalid data found when processing input

[out#0/webm @ 0000023d6d784980] Error muxing a packet

I don't know what to do to fix this, thanks for your help

Translated ::

Erreur de téléchargement : 
[libaom-av1 @ 0000023d6d6eb440] Échec du décodage de la trame : Bitstream non pris en charge par ce décodeur

    Dernier message répété 30 fois

[vost#0:1/copy @ 0000023d6d765140] Erreur lors de la soumission d'un paquet au multiplexeur : données non valides trouvées lors du traitement de l'entrée

[out#0/webm @ 0000023d6d784980] Erreur lors du multiplexage d'un paquet.

Je ne sais pas quoi faire pour arranger ça, merci pour votre aide

________________________________________

### Addon & CoApp version and details

Mardi 28.05.24.
Bonjour ;

Je n'y comprends plus rie, depuis pas mal de temps, je n'arrive plus à télécharger avec Vidéo Download Internet de Firefox, en effet, j'ai toujours cette erreur à chaque fois ::

SideDownload error: 
[libaom-av1 @ 0000023d6d6eb440] Failed to decode frame: Bitstream not supported by this decoder

    Last message repeated 30 times

[vost#0:1/copy @ 0000023d6d765140] Error submitting a packet to the muxer: Invalid data found when processing input

[out#0/webm @ 0000023d6d784980] Error muxing a packet

I don't know what to do to fix this, thanks for your help

Translated ::

Erreur de téléchargement : 
[libaom-av1 @ 0000023d6d6eb440] Échec du décodage de la trame : Bitstream non pris en charge par ce décodeur

    Dernier message répété 30 fois

[vost#0:1/copy @ 0000023d6d765140] Erreur lors de la soumission d'un paquet au multiplexeur : données non valides trouvées lors du traitement de l'entrée

[out#0/webm @ 0000023d6d784980] Erreur lors du multiplexage d'un paquet.

Je ne sais pas quoi faire pour arranger ça, merci pour votre aide

________________________________________

**Comment by GrampaWildWilly** [comment]
On a besoin de connaître ton système d'exploitation avec la version, ton navigateur avec la version, ta version de VDH, ta version du CoApp.  On a aussi besoin d'un URL d'une page particulière où se trouve une vidéo.  Ne donne pas le nom d'un site web.  Donne l'URL exact d'une page où tu as de la misère à faire marcher VDH comme il faut.  Ne dis pas que c'est partout et n'importe où.  Au moins, donne un exemple représentatif d'une page qui montre les symptômes non voulus.

**Comment by Franck-DUBUS** [comment]
Jeudi 30.05.24.



Bonjour,

Merci pour le retour ! Je suis sous Windows 10 Pro V.22H2 Build:
19045.4412, navigateur: Firefox v.126.0.1, version de VDH : Video
DownloadHelper 9.0.2.3 (stable)   Appli compagnon trouvée: *VdhCoApp 2.0.19*

Binaire de l'appli compagnon: *C:\Program Files\DownloadHelper
CoApp\vdhcoapp.exe *

Désolé, mais je ne peux pas donner l'URL exacte su site en question,
pardonnez ma franchise, mais tout ce que je peux dire, c'et qu'avant ça
n'était pas très brillant, mais là, c'est vraiment catastrophique !!



A toute fin utile ::

Haut du formulaire

Video DownloadHelper⊗

Processeur de téléchargement

Bas du formulaire

*Application réalisant le téléchargement*  ::  *Navigateur *

Dossier de téléchargement par défaut  :: * C:\Users\Jack\dwhelper*

*Seulement utilisé avec l'application compagnon comme processeur de
téléchargement*


[image: image.png]

Merci pour votre aide.
Cdt



Le mar. 28 mai 2024 à 13:16, GrampaWildWilly ***@***.***> a
écrit :
